gto/cli.py

```diff
@@ -218,7 +218,7 @@ def GTOGroupSection(section):
     ".",
     "-r",
     "--repo",
-    help="Repository to use",
+    help="Repository to use (experimental support for remote repos)",
```

Comment:
```suggestion
    help="Repository to use (+ experimental support for remote repos)",
```

Reply:
btw, we say it everywhere when we have `--commit` and `--push`. Do we need to mention it here as well? IIRC, the biggest part of problems could arise is when you have a local repo and do something (i.e. non-consistency between local and remote). Not when you use the remote repo in `--repo`, cause then the problem frequency should be negligible. 
thus, should we add "experimental" word here at all? Maybe just "Repository to use (local or remote)" or something?

Reply:
Perhaps I am missing something: will the "+" do something, or the text will be rendered exactly as written: `(+ experimental support ...`?
I have never seen this convention before.

Reply:
Yes, never seen this convention as well. Just thought one may assume we support remotes only, that's why I suggested `+`. But anyway, would you be ok with removing `experimental` word at all here?

Reply:
oh ok, that's what you meant :)
I think it's a bit unclear...

No problem about removing the `experimental`, but then we should remove that word also from the `commit` and `push` options, right? Shall I do it as part of this PR?

What about renaming the whole into `Local or remote repository` (`to use` perhaps is obvious and can be omitted)?

Reply:
`Local or remote repository` is fine to me. As I said above, working with remote repos look robust to me (it shouldn't cause errors 99% of the time). But working with local repos + `--push`/`--commit` looks unstable (there could be some errors - that's why I suggested adding `experimental` in first place). So I'm for keeping `experimental` there, and not using `experimental` here for `--repo` option.

Reply:
Do you agree / disagree? :)

Reply:
I understand your concern, so I agree.

Changed: https://github.com/iterative/gto/pull/297/commits/0097ea68c299ec4e85091c25729e89e368762e38